apps/src/code-studio/appRedux.js

```diff
@@ -0,0 +1,31 @@
+const SET_APP_LOAD_STARTED = 'header/SET_APP_LOAD_STARTED';
+const SET_APP_LOADED = 'header/SET_APP_LOADED';
+
+const initialState = {
+  appLoading: false,
```

Comment:
i know you just moved this state from its previous location, but it looks like it doesn't exist anywhere. in redux, we only ever set `appLoadStarted`, which is used in HeaderMiddle.jsx -- we never set or access `appLoading` anywhere. i think we should update this `initialState` -- it was likely an artifact or a renaming mistake

Reply:
This was a great catch! Will change appLoading to appLoadStarted.